### DISCUSSION TITLE
Can I self-host if domain and hosting is by porkbun.com?

### DISCUSSION THREAD

**Discussion by proateen** [plausible/analytics#916] [Self-Hosted Support] [answered]
I've bought a domain from porkbun.com and purchased their monthly static hosting service. Will I be able selfhost Plausible.io in this way?
All the features I have access to:
![image](https://user-images.githubusercontent.com/52085315/114300676-65a6f580-9ab9-11eb-8122-d20ad8add6b5.png)

**Comment by Vigasaurus** [comment, marked as answer]
No, static hosting is just that, **static**. The Plausible server is plenty dynamic with 2 databases, and consistent run time of accepting events. Porkbun's (or any provider's, for that matter) static hosting will just let you host basic HTML/CSS/JS sites, not any which require a backend, databases etc (as plausible does). 

Your best bet will likely just be to use the hosted version of Plausible directly, but if you really do want to self-host, you'll need to ensure the provider you use gives you access to run Docker apps for the most ease (and generalized ability) of use.
